vendor/bat-native-ledger/src/bat/ledger/internal/common/security_helper.cc

```diff
@@ -4,14 +4,35 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include <openssl/base64.h>
+#include <openssl/digest.h>
```

Comment:
Given that we're moving code from `bat_helper.cc` to this file, there might be includes in `bat_helper.cc` that can be removed, but it's hard for me to tell.

Reply:
we need all includes, this one specific is needed for `EVP_sha512()`